From 861e933a494c12f00160091375faec9357fb4190 Mon Sep 17 00:00:00 2001
From: Yo-Jung Lin <leo.lin@canonical.com>
Date: Thu, 16 Nov 2023 14:05:40 +0800
Subject: [PATCH] hwdb: Mark Dell platform accel sensor location to base

Dell would like to disable screen rotation for the platform eternally.
Mark the aceel sensor location base to disable it.

(cherry picked from commit eb8258c1bcc4e6c14246d828832f27dfb083cda9)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index c17f57ace7..75efc3c190 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -383,6 +383,15 @@ sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C02:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C40:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C41:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C42:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CBD:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CBF:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC0:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC1:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC3:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC4:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC5:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC7:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0CC8:*
  ACCEL_LOCATION=base
 
 # Dell Venue 8 Pro 3845
